src/components/App.css

```diff
@@ -11,3 +11,7 @@ h3, .cerise #content h3 {
 .pagination .disabled, .pagination .disabled a {
   color: #999 !important;
 }
+
+h1, h2, h3, h4, h5, h6 {
+  scroll-margin-top: 60px
```

Comment:
Would you not want to put this on the whole element that is sibling to methone (if that's possible)? I'm guessing methones height is hardcoded to 60px btw?

Reply:
Possibly better now?
Methone is from what i can see hardcoded to 50px. I started with that but tried around a bit and thought 60px looked nicer.

Reply:
Ah, I realised I missparsed what you ment. CSS isn't my strong suit, but from what I know and a little testingI don't think it is possible using this rule. The rule seems to have to be applied to the exact target of a link. 

There's probably some way to fix it by changing how methone is implemented, or changing some margins and paddings on the wrapping divs (would probably be uglier tbh). But this works and this rule seems to be designed to solve this exact problem, so I think this is a good enough solution.